region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -738,6 +738,36 @@
         "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
       ]
     },
+    {
+      "link": [5, 1],
+      "name": "G-Mode Setup - Get Hit By Mella (Long Manipulaton)",
```

Comment:
I wasn't able to test this one, so I just copied the strat that went to the item. Can someone else test it?